crates/bevy_window/src/window.rs

```diff
@@ -869,6 +870,70 @@ impl From<DVec2> for WindowResolution {
     }
 }
 
+/// Common screen resolutions.
+///
+/// These resolutions are common resolutions that are more than likely going to be used, using the
+/// common name. This keeps developers from having to remember each resolution they wish to use, along
+/// with makes things simpler for implementing menus.
+#[derive(Debug, Clone, Copy, Eq, PartialEq, Hash)]
+#[non_exhaustive]
+pub enum CommonScreenResolution {
+    /// 640 x 360
+    R360p,
+
+    /// 1280 x 720
+    R720p,
+
+    /// 1920 x 1080
+    R1080p,
+
+    /// 2560 x 1440
+    R2k,
```

Comment:
Calling QHD resolution 2K is a misnomer. 2K is closer to to 1080p (FullHD) than QHD.